### ISSUE TITLE
[CLI]: WandbTracer is not working with langchain v0.0.169+

### ISSUE THREAD

**Issue by kde713** [wandb/wandb#5557]
### Describe the bug

`WandbTracer.__init__(...)` does not work with langchain v0.0.169 and above.
Change from https://github.com/hwchase17/langchain/pull/4549 breaks internal `TracerSession` initialization.

```python
from wandb.integration.langchain import WandbTracer

tracer = WandbTracer({"project": "project-name-here"})
```

```shell
Traceback (most recent call last):
  File ".../wandb-langchain.py", line 36, in <module>
    main()
  File ".../wandb-langchain.py", line 12, in main
    WandbTracer({"project": "onboarding-prompt"})
  File "/../venv/lib/python3.9/site-packages/wandb/integration/langchain/wandb_tracer.py", line 140, in __init__
    self.session = self.load_session("")
  File ".../venv/lib/python3.9/site-packages/wandb/integration/langchain/wandb_tracer.py", line 225, in load_session
    self._session = TracerSession(id=1)
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 2 validation errors for TracerSession
tenant_id
  field required (type=value_error.missing)
id
  value is not a valid uuid (type=type_error.uuid)
```

### Environment

WandB version: `0.15.2`

OS: `macOS 13.2.1 22D68 arm64`

Python version: `3.9.6`

Versions of relevant libraries:
```
langchain==0.0.170
```

**Comment by rsanandres-wandb** [comment]
Hello @kde713 !

It looks like our team is working on the issue already but I will be making a bug report for thoroughness. Thank you for reporting this!

**Comment by kde713** [comment]
@rsanandres-wandb Seems like this issue was resolved in v0.15.3 with PR #5558. Is it okay to close this?
